README.md

```diff
@@ -6,13 +6,252 @@ An Ansible role which installs and configures [Patroni](https://github.com/zalan
+- `patroni_superuser_password`: PostgreSQL DB password for master (**Default:** `supersecretpostgrespasswd`)
+- `patroni_dcs`: Which DCS is configured (**Default:** `etcd`)
+- `patroni_dcs_exists`: Does a DCS exist (**Default:** `true`)
+
+### Global/Universal
+
+Read more here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#global-universal
+
+- `patroni_scope`: Scope of the Patroni cluster (**Default:** `main`)
+- `patroni_namespace`: Namespace of the Patroni cluster (**Default:** `/service/`)
+- `patroni_name`: Name of the Patroni node (**Default:** `"{{ inventory_hostname }}"`)
+
+### Log
+
+Read more here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#log
+
+- `patroni_log_destination`: The  (**Default:** `stderr`)
+- `patroni_log_level`: Sets the general logging level. Default value is INFO (see https://docs.python.org/3.6/library/logging.html#levels) (**Default:** `INFO`)
+- `patroni_log_format`: Sets the log formatting string. Default value is %(asctime)s %(levelname)s: %(message)s (see https://docs.python.org/3.6/library/logging.html#logrecord-attributes) (**Default:** `"%(asctime)s %(levelname)s: %(message)s"`)
+- `patroni_log_dateformat`: Sets the datetime formatting string. (see https://docs.python.org/3.6/library/logging.html#logging.Formatter.formatTime) (**Default:** `""`)
+- `patroni_log_max_queue_size`: Patroni is using two-step logging. Log records are written into the in-memory queue and there is a separate thread which pulls them from the queue and writes to stderr or file. The maximum size of the internal queue is limited by default by 1000 records, which is enough to keep logs for the past 1h20m. (**Default:** `1000`)
+- `patroni_log_dir`: Directory to write application logs to. The directory must exist and be writable by the user executing Patroni. If you set this value, the application will retain 4 25MB logs by default. You can tune those retention values with `patroni_log_file_num` and `patroni_log_file_size` (**Default:** `/var/log/patroni`)
+- `patroni_log_file_num`: The number of application logs to retain (**Default:** `4`)
+- `patroni_log_file_size`: Size of patroni.log file (in bytes) that triggers a log rolling (**Default:** `25000000`)
+- `patroni_log_loggers`: This section allows redefining logging level per python module. **Defaults:**
+  - `- { module: "patroni.postmaster", level: "WARNING" }`
+  - `- { module: "urllib3", level: "DEBUG" }`
+
+### REST API
+
+Read more here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#rest-api
+- `patroni_restapi_port`: The Port for the `patroni_restapi_port`. (**Default:** `8008`)
+- `patroni_restapi_connect_address`: IP address (or hostname) and port, to access the Patroni’s REST API. (**Default:** `"{{ ansible_host }}:{{ patroni_restapi_port }}"`)
+- `patroni_restapi_listen`: IP address (or hostname) and port that Patroni will listen to for the REST API. (*Default* `"0.0.0.0:{{ patroni_restapi_port }}"`)
```

Comment:
My assumption is that the `patronictl` command is being executed on the nodes with the local IP address. Therefore, it is necessary to also set a localhost or local IP address for the API so it will also listen to it. The connect_address is the public address that must be reachable for all nodes.

So they are not exactly the same, but it's better described in the documentation here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#rest-api

Reply:
I checked our dev again and there is not a source address required for other nodes to connect to Patroni. But I think for localhost it is. It's possible that the other nodes certificate with certificates, while `patronictl` doesn't. I will research it next week.